staging/src/k8s.io/client-go/util/workqueue/queue_test.go

```diff
@@ -43,7 +43,7 @@ func TestBasic(t *testing.T) {
 	}
 	for _, test := range tests {
 		// If something is seriously wrong this test will never complete.
-
+		test := test
```

Comment:
I would prefer a different name for local variable here to avoid the confusion. Maybe `testcase` or something like that.

Reply:
actually, shadowing the name minimizes changes of bugs where we reference the loop variable incorrectly

Reply:
Good point, Jordan. I'm fine with that choice.

Reply:
we should use subtests with t.Run, that is much better solution

Reply:
if the subtests run in parallel, you'd still need `test := test`. if they don't run in parallel, then no change is needed here at all